### PR TITLE
Adds death sounds for the SPS

### PR DESCRIPTION
**Pull request by ghost**
ripped from hl2 combine death sounds.
this is the greatest feature of all time

https://youtu.be/52OFAmirOfU

:cl:
* soundadd: adds sound for SPS when someone dies.

### CODE REVIEW

**Comment by ghost** [comment]
I can make this better

**Comment by ghost** [comment]
I made this better

**Comment by ghost** [comment]
will post video of result soon.

**Comment by ghost** [comment]
SEC13 beeps: Warning! SPS 'SEC13' has detected the death of their wearer at Medbay (243, 263, 1).
https://youtu.be/52OFAmirOfU

**Comment by DeityLink** [comment]
YES

**Comment by SonixApache** [comment]
I like it. Dunno about balance, though.
The deathscream is definitely weird, maybe cut that part out. Was expecting the HL2 death sound.

**Comment by ghost** [comment]
@SonixApache That's not a scream, I've always heard it as them flatlining

**Comment by SonixApache** [comment]
It starts with a garbled `GUH` of sorts. After that's a flatline. I'd say cut those, and just keep the report part.

**Comment by ghost** [comment]
Eh, yeah; I can cut out the GUH but I think the flatline is good.

**Comment by SonixApache** [comment]
On a different note, maybe prefix your number sound files with something.

**Comment by iFlashYou** [comment]
I hope you cant hear this trough walls or?

File: code/modules/telesci/gps.dm [hunk 1]
```diff
@@ -169,21 +169,74 @@ var/list/SPS_list = list()
...
+			playsound(src, 'sound/items/attention.wav',100, 0,channel = sound_channel,wait = TRUE)
+			playsound(src, 'sound/items/_comma.wav',100, 0,channel = sound_channel,wait = TRUE)
+
+		if(prob(25) && dead) // 25% chance if dead, 0% chance if stripped
+			playsound(src, 'sound/items/unitdeserviced.wav',100, 0,channel = sound_channel,wait = TRUE)
+			playsound(src, 'sound/items/_comma.wav',100, 0,channel = sound_channel,wait = TRUE)
+		else if(prob(33)) 	// 25% chance if dead, 50% chance if stripped
+			playsound(src, 'sound/items/lostbiosignalforunit.wav',100, 0,channel = sound_channel,wait = TRUE)
+			playsound(src, 'sound/items/_comma.wav',100, 0,channel = sound_channel,wait = TRUE)
+			playnum(rand(1,9),pos,sound_channel)
```
Comment (Kurfursten) [inline code/modules/telesci/gps.dm]: you could make this match the index in the SPS list

**Review by MadmanMartian — COMMENTED** [review]
See about converting the .wav files to .ogg for filesize saving.

**Comment by Probe1** [comment]
They're 15-100kb in size which is in line with the existing sound files.  If you want it for file type consistency that's fine but we're already sending sounds in the size range so it's not a problem.  It's rarely a problem when admins send 3mb oggs to the entire serb for that matter.

**Comment by ghost** [comment]
heck

**Comment by NigglyWiggly** [comment]
This is fucking horrible, Celt.
It sounds so fucking bad ingame and it fucking destroys immulsions

**Comment by Probe1** [comment]
good thing you don't play to have your immulsions ruined 😉

**Comment by NigglyWiggly** [comment]
too bad I've been hearing it every time I'm on and it's horrible

**Pull request merged**